integration-tests/gradle/src/test/java/io/quarkus/gradle/FastJarFormatWorksTest.java

```diff
@@ -20,6 +31,53 @@ public void testFastJarFormatWorks() throws Exception {
         assertThat(quarkusApp).exists();
         Path jar = quarkusApp.resolve("quarkus-run.jar");
         assertThat(jar).exists();
+
+        File output = new File(projectDir, "build/output.log");
+        output.createNewFile();
+        Process process = launch(jar, output);
+        try {
+            // Wait until server up
+            dumpFileContentOnFailure(() -> {
+                await()
+                        .pollDelay(1, TimeUnit.SECONDS)
+                        .atMost(1, TimeUnit.MINUTES).until(() -> DevModeTestUtils.getHttpResponse("/hello", 200));
+                return null;
+            }, output, ConditionTimeoutException.class);
+
+            String logs = FileUtils.readFileToString(output, "UTF-8");
+
+            assertThatOutputWorksCorrectly(logs);
+            // test that the application name and version are properly set
+
+            getHttpResponse("/hello");
+            assertThat(getHttpResponse("/hello").equals("hello"));
+        } finally {
+            process.destroy();
+        }
+    }
+
+    static void assertThatOutputWorksCorrectly(String logs) {
+        assertThat(logs.isEmpty()).isFalse();
+        String infoLogLevel = "INFO";
+        assertThat(logs.contains(infoLogLevel)).isTrue();
+        String basic = "[io.quarkus] (main)";
```

Comment:
No need to test this I would say

Reply:
But you said to test any string which is always present while removing the regex

Reply:
Yeah, what am I saying is that this test doesn't make sense because you have `assertThat(logs.contains("cdi, resteasy")).isTrue();`

Reply:
Okkay